Sources/_StringProcessing/ByteCodeGen.swift

```diff
@@ -16,6 +16,7 @@ import Swift
 
 extension Compiler {
   struct ByteCodeGen {
+    var reverse = false
```

Comment:
@natecook1000 what do you think about modeling reverse as an option? Do you think that would clean up the implementation?